web-client/integration-tests/journey/docketClerkCreatesATrialSession.js

```diff
@@ -132,5 +134,6 @@ export const docketClerkCreatesATrialSession = (
     expect(lastCreatedTrialSessionId).toBeDefined();
 
     cerebralTest.lastCreatedTrialSessionId = lastCreatedTrialSessionId;
+    cerebralTest.trialSessionId = cerebralTest.lastCreatedTrialSessionId;
```

Comment:
Are both of these needed? I think they are the same value?

Reply:
I believe `lastCreatedTrialSessionId` is used in a separate journey file that calls this! However we need `trialSessionId` to be set in state because of yet another file we call from `web-client/integration-tests/trialSessionChangeToRemoteProceedingJourney.test.js` that expects `cerebralTest.trialSessionId` to be set. I can def make a devex to DRY up trial session related integration tests, if its needed?